src/content/docs/en/guides/images.mdx

```diff
@@ -481,3 +480,95 @@ export default defineConfig({
 ## Community Integrations
 
 There are several third-party [community image integrations](https://astro.build/integrations?search=images) for optimizing and working with images in your Astro project.
+
+
+## Inline `.svg` files in the HTML (experimental)
+
+This feature allows you to import SVG files directly into your Astro project. By default, Astro will inline the SVG content into your HTML output.
+
+To enable this feature, set `experimental.svg` to `true` in your Astro config:
+
+ ```js
+{
+  experimental: {
+    svg: true,
+  },
+}
+```
+
+To use this feature, import an SVG file to use as an Astro component, passing any common SVG attributes. Astro also provides a `size` attribute to set equal `height` and `width` properties. This process assigns an `<svg>` HTML tag to a component, and requires the same conventions (e.g. capitalization) as when [using dynamic tags](/en/basics/astro-syntax/#dynamic-tags).
```

Comment:
```suggestion
To use this feature, reference the default import of any local `.svg` file. Since this import is treated as an Astro component, it requires using the same conventions (e.g. capitalization) as when [using dynamic tags](/en/basics/astro-syntax/#dynamic-tags).
```

Simplified for clarity and avoids repetition.

Reply:
Love it!